storage/persistent-storage/persistent-storage-efs.adoc

```diff
@@ -0,0 +1,46 @@
+[id="persistent-storage-using-aws-efs"]
+= Persistent storage using AWS Elastic File System
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-efs
+
+toc::[]
+
+{product-title} allows use of AWS Elastic File System volumes (EFS). You can
+provision your {product-title} cluster with persistent storage using AWS EC2.
+Some familiarity with Kubernetes and AWS is assumed.
+
+:FeatureName: Elastic File System
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+The Kubernetes persistent volume framework allows administrators to provision a
+cluster with persistent storage and gives users a way to request those
+resources without having any knowledge of the underlying infrastructure.
+AWS Elastic Block Store volumes can be provisioned dynamically.
+Persistent volumes are not bound to a single project or namespace; they can be
+shared across the {product-title} cluster.
+Persistent volume claims are specific to a project or namespace and can be
+requested by users.
+
+[IMPORTANT]
+====
+Amazon security groups must be configured to allow inbound NFS traffic
+from the EFS volume's security group. The EFS volume must be configured to
+allow incoming SSH traffic from any host.
+====
+
+.Additional references
```

Comment:
Do you want this before your topic body?

Reply:
I think listing it at the end of the opening makes sense, providing it immediately before we begin discussing how to accomplish the process of using EFS.